manifests/v1beta1/katib-controller/base/katib-controller-deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
```

Comment:
Should we move all of the YAMLs from the `manifests/v1beta1/katib-controller/base` to the `/manifests/v1beta1/components/<appropriate-component-name/` ?
Since we left the old structure for the manifests to not brake `kfdef` configs before we transfer to the v3 manifests ?

Reply:
I agree! I was suspecting something like this happened.